Infrastructure/Models/User.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
I think these imports are done automatically by VS, but most (if not all) are unused, so please remove :) Same in all other files.